apollo-at-link/src/link_spec_definition.rs

```diff
@@ -0,0 +1,57 @@
+use crate::spec::{Identity, Url, Version};
+use crate::spec_definition::{SpecDefinition, SpecDefinitions};
+use lazy_static::lazy_static;
+
+pub struct LinkSpecDefinition {
+    url: Url,
+    minimum_federation_version: Option<Version>,
+}
+
+impl LinkSpecDefinition {
+    pub fn new(
+        version: Version,
+        minimum_federation_version: Option<Version>,
+        identity: Identity,
+    ) -> Self {
+        Self {
+            url: Url { identity, version },
+            minimum_federation_version,
+        }
+    }
+}
+
+impl SpecDefinition for LinkSpecDefinition {
+    fn url(&self) -> &Url {
+        &self.url
+    }
+
+    fn minimum_federation_version(&self) -> Option<&Version> {
+        self.minimum_federation_version.as_ref()
+    }
+}
+
+lazy_static! {
```

Comment:
I am not 100% sure on the current status of `const` for this use case (might already work, or it might be still in the works),  buttttt with recent `const` developments we basically try to use `const` instead of a lot of `lazy_static!` calls we used to do. You may have to refactor this just a bit to get it to work properly with const.